PayPal/src/main/java/com/braintreepayments/api/PayPalLineItem.java

```diff
@@ -137,6 +159,33 @@ public void setUrl(@NonNull String url) {
         this.url = url;
     }
 
+    /**
+     * The image URL to product information.
+     *
+     * @param imageURL The image URL with additional information.
+     */
+    public void setImageUrl(@NonNull String imageUrl) {
+        this.imageUrl = imageUrl;
+    }
+
+    /**
+     * UPC code of the item.
+     *
+     * @param upcCode The UPC code.
+     */
+    public void setUpcCode(@NonNull String upcCode) {
+        this.upcCode = upcCode;
+    }
+
+    /**
+     * UPC type of the item.
+     *
+     * @param upcType The UPC type.
+     */
+    public void setUpcType(@NonNull String upcType) {
```

Comment:
```suggestion
    public void setUpcType(@Nullable String upcType) {
```
I'd actually go for `@Nullable` for these properties to align the getter/setter annotations. `upcType: String?` makes sense since it's initially `null`.

Reply:
Currently all the existing properties have @NonNull for setters and @Nullable for getters, as these are not required fields and merchant may or may not use setter methods. We didn't see any issues so far.

Reply:
I think that must have been an oversight when those properties were added. When these annotations don't match up, the Kotlin property setters don't seem to work as expected (ex: `lineItem.upcType = "some type"` won't work in Kotlin, they would have to set `lineItem.setUpcType("some type")`), which isn't the ideal Kotlin integration experience. Although I'm not certain if newer versions of Kotlin have resolved this issue.

Reply:
Oh cool yeah if it does work in newer versions of Kotlin i'm 👍 . We should check though. The way it's written now makes total sense to me, but I have seen Kotlin get weird when the annotations aren't aligned.

Reply:
I just confirmed with the v5 branch that uses the latest Kotlin that this issue still exists 😞 - I created a ticket to update the annotations in v5 where they don't currently align, but we should add these new properties with matching annotations.

Reply:
Updated setter method annotation from `@NonNull` to `@nullable` to keep it consistent with getter method annotations